pkg/scheduler/job.go

```diff
@@ -118,10 +113,10 @@ func (m *metricJob) refreshMetrics() {
 	}
 
 	m.syncDone.Do(func() {
-		klog.V(1).Infof(
-			"First sync successful from %s / %s",
-			m.GetClient().GetConfiguration().Name,
-			m.GetClient().GetConfiguration().ClientConfig.Host,
+		logger.V(1).Info(
```

Comment:
It no longer seems to be logged on startup while it is the case with the code in `main`:

```
I0905 09:07:36.967885    8074 job.go:121] First sync successful from k8s-region-observability-cluster / ${HPA_ELASTICSEARCH_HOST}
```

I'm using `--v=2` as mentioned in the [README](https://github.com/elastic/elasticsearch-k8s-metrics-adapter/blob/main/README.md#logs) (which we should maybe update with this PR).

Reply:
I see I broke the way the `v` flag was reused to configure the logger. Now it only uses the 0 default level and does not take into account what is defined in `--v`. I've mixed feeling between restoring this and deleting most of the `V(x)` loggers, converting a few and so in the end keeping only `Info` and `Error`.

Reply:
Fixed in 5018ed9.